packages/trip-form/src/TripOptions/ModeRow.tsx

```diff
@@ -0,0 +1,126 @@
+import coreUtils from "@opentripplanner/core-utils";
+import React, { useEffect, useRef } from "react";
+
+import {
+  categoryIsActive,
+  getCategoryPrimaryMode,
+  getSelectedModes
+} from "./util";
+import { Modes, QueryParams } from "./types";
+import * as S from "./styled";
+
+import Checkbox from "./Checkbox";
+
+const ModeRow = ({
+  onQueryParamChange,
+  queryParams,
+  queryParamOverrides,
+  supportedModes,
+  SimpleModeIcon
+}: {
+  onQueryParamChange(paramsToUpdate: QueryParams, categoryId?: string): void;
+  queryParams: QueryParams;
+  queryParamOverrides: { [key: string]: QueryParams };
+  supportedModes: Modes;
+  SimpleModeIcon?: React.FunctionComponent<{ mode: string }>;
+}): React.ReactElement => {
+  const { categories } = supportedModes;
+  const selectedModes = getSelectedModes(queryParams);
+  const selectedTransit = selectedModes.filter(coreUtils.itinerary.isTransit);
+  const hasTransit = selectedTransit.length > 0;
+  const selectedTransitString = selectedTransit.join(",") || "TRANSIT";
+
+  // Scroll to active mode on initial render
+  // This ref is attached to every active mode checkbox
+  const initialRenderRef = useRef() as React.MutableRefObject<HTMLInputElement>;
+
+  useEffect(() => {
+    initialRenderRef?.current?.scrollIntoView({
```

Comment:
This seems to work fine elsewhere, but when I run the jest snapshot test for the component that is the wrapper on TripOptions in trimet.org I'm getting the following error:

```
console.error node_modules/jsdom/lib/jsdom/virtual-console.js:29
    Error: Uncaught [TypeError: _initialRenderRef$cur.scrollIntoView is not a function]
        at reportException (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\helpers\runtime-script-errors.js:66:24)
        at invokeEventListeners (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:209:9)
        at HTMLUnknownElementImpl._dispatch (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\events\EventTarget-imp
l.js:119:9)
        at HTMLUnknownElementImpl.dispatchEvent (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\events\EventTarget
-impl.js:82:17)
        at HTMLUnknownElementImpl.dispatchEvent (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\nodes\HTMLElement-
impl.js:30:27)
        at HTMLUnknownElement.dispatchEvent (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\generated\EventTarget.
js:157:21)
        at Object.invokeGuardedCallbackDev (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:237:16)
        at invokeGuardedCallback (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:292:31)
        at flushPassiveEffectsImpl (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:22853:9)
        at unstable_runWithPriority (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\node_modules\scheduler\cjs\scheduler.develo
pment.js:653:12) TypeError: _initialRenderRef$cur.scrollIntoView is not a function
        at C:\Users\humphrig\projects\trimet.org\node_modules\@opentripplanner\trip-form\src\TripOptions\ModeRow.tsx:38:32
        at commitHookEffectListMount (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:19731:26)
        at commitPassiveHookEffects (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:19769:11)
        at HTMLUnknownElement.callCallback (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:188:14)
        at invokeEventListeners (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:193:27)
        at HTMLUnknownElementImpl._dispatch (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\events\EventTarget-imp
l.js:119:9)
        at HTMLUnknownElementImpl.dispatchEvent (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\events\EventTarget
-impl.js:82:17)
        at HTMLUnknownElementImpl.dispatchEvent (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\nodes\HTMLElement-
impl.js:30:27)
        at HTMLUnknownElement.dispatchEvent (C:\Users\humphrig\projects\trimet.org\node_modules\jsdom\lib\jsdom\living\generated\EventTarget.
js:157:21)
        at Object.invokeGuardedCallbackDev (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:237:16)
        at invokeGuardedCallback (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:292:31)
        at flushPassiveEffectsImpl (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:22853:9)
        at unstable_runWithPriority (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\node_modules\scheduler\cjs\scheduler.develo
pment.js:653:12)
        at runWithPriority$1 (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:11039:10)
        at flushPassiveEffects (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom.development.js:22820:12)
        at Object.<anonymous>.flushWork (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom-test-utils.development.js
:876:10)
        at act (C:\Users\humphrig\projects\trimet.org\node_modules\react-dom\cjs\react-dom-test-utils.development.js:994:9)
        at render (C:\Users\humphrig\projects\trimet.org\node_modules\@testing-library\react\dist\pure.js:82:26)
        at Object.<anonymous> (C:\Users\humphrig\projects\trimet.org\src\trip-planner\components\TripSettings\TripSettings.test.js:18:28)
        at Object.asyncJestTest (C:\Users\humphrig\projects\trimet.org\node_modules\jest-jasmine2\build\jasmineAsyncInstall.js:102:37)
        at C:\Users\humphrig\projects\trimet.org\node_modules\jest-jasmine2\build\queueRunner.js:43:12
        at new Promise (<anonymous>)
        at mapper (C:\Users\humphrig\projects\trimet.org\node_modules\jest-jasmine2\build\queueRunner.js:26:19)
        at C:\Users\humphrig\projects\trimet.org\node_modules\jest-jasmine2\build\queueRunner.js:73:41
        at processTicksAndRejections (internal/process/task_queues.js:95:5)

  console.error node_modules/react-dom/cjs/react-dom.development.js:19527
    The above error occurred in the <ModeRow> component:
        in ModeRow (created by TripOptions)
        in div (created by styled__TripOptionsContainer)
        in styled__TripOptionsContainer (created by TripOptions)
        in TripOptions (created by Styled(TripOptions))
        in Styled(TripOptions) (created by TripSettings)
        in div (created by styled.div)
        in styled.div (created by TripSettings)
        in TripSettings
        in IntlProvider (created by IntlShim)
        in IntlShim
        in Unknown
    
    Consider adding an error boundary to your tree to customize error handling behavior.
    Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
```

Reply:
This should be fixed now in the latest alpha, please let me know if the issue is fixed for you!